3-networks-hub-and-spoke/modules/base_shared_vpc/main.tf

```diff
@@ -126,7 +126,10 @@ module "region1_router1" {
   bgp = {
     asn                  = var.bgp_asn_subnet
     advertised_groups    = ["ALL_SUBNETS"]
-    advertised_ip_ranges = [{ range = local.private_googleapis_cidr }]
+    advertised_ip_ranges = [
+      { range = local.private_googleapis_cidr },
+      { range = "35.199.192.0/19" }
```

Comment:
`{ range = "35.199.192.0/19" }` is conditional on the network being the one that  will access the dns server in the on-prem infrastructure